src/patterns/DetailList/DetailList.tsx

```diff
@@ -31,8 +30,9 @@ export interface DetailListProps extends ComponentProps<'dl'> {
  *
  */
 const DetailList = forwardRef<HTMLDListElement, DetailListProps>(
-  ({ alignment, className, children, hasSeparators, ...props }, ref) => {
+  ({ alignment = DetailListAlignment.columns, className, children, hasSeparators = false, ...props }, ref) => {
```

Comment:
I think making the other Alignment the default is better as that seems to be more common in our current Figmas